### DISCUSSION TITLE
Short audio clips won't load correctly

### DISCUSSION THREAD

**Discussion by samfcheng** [katspaugh/wavesurfer.js#3610] [Q&A]
Using SvelteKit and having this issue where short audio clips will not display
![image](https://github.com/katspaugh/wavesurfer.js/assets/51756184/98ff00a5-be57-4ccc-8f0e-329d8872d576)

Here's how I'm loading the waveform:
![image](https://github.com/katspaugh/wavesurfer.js/assets/51756184/8e87ad88-8c65-4fcc-9b74-84be96502559)

[This other discussion](https://github.com/katspaugh/wavesurfer.js/discussions/3521) mentions that WAV files don't work if they're on the shorter end, but I just tried switching to MP3s and still having that same issue.

Any ideas on why this is happening or how it could be fixed?

Edit: did some testing and it seems to me that if the audio won't fill the entire container, it won't work. I could fix this by upping the pixels per second, but it's not a fix I like.

**Comment by katspaugh** [comment]
Hmm it’s supposed to fill the container even with a lower px-per-second. Can you upload a sample file for me to test?
